File: 1.C/1.c
/*1. Diferenciar dentre 3 números, qual é o maior e qual é o menor*/

#include <stdio.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL, "Portuguese");

    int ENTRADA[3];
    int i, MAIOR, MENOR;

    for (i = 0; i < 3; i++)
    {
        printf("%d.Número: ", i + 1);
        scanf("%d", &ENTRADA[i]);
    }

    MENOR = ENTRADA[0];
    MAIOR = ENTRADA[0];

    for (i = 0; i < 3; i++)
    {
        if (ENTRADA[i] > MAIOR)
        {
            MAIOR = ENTRADA[i];
        }
        if (ENTRADA[i] < MENOR)
        {
            MENOR = ENTRADA[i];
        }
    }

    printf("MENOR: %d\n", MENOR);
    printf("MAIOR: %d\n", MAIOR);
}

File: 2.JS/1.js
/*24. Diferenciar dentre 3 números, qual é o maior e qual é o menor*/

let entrada = [5, 2, 8];
let menor = entrada[0];
let maior = entrada[0];
let tamanho = entrada.length;

for (let i = 0; i < tamanho; i++) {
    if (entrada[i] > maior) {
        maior = entrada[i];
    }
    if (entrada[i] < menor) {
        menor = entrada[i];
    }
}

console.log("MENOR: " + menor);
console.log("MAIOR: " + maior);
